backend/api/v2beta1/experiment.proto

```diff
@@ -87,9 +85,15 @@ message Experiment {
   // Optional input field. Specify the namespace this experiment belongs to.
   string namespace = 5;
 
+  // Describes whether an entity is available or archived.
   enum StorageState {
+    // Default state. This state in not used
     STORAGESTATE_UNSPECIFIED = 0;
+
+    // Entity is available.
     AVAILABLE = 1;
+
+    // Entity is archived.
     ARCHIVED = 2;
   }
 
```

Comment:
Should we add `Error error = 7;`?

Reply:
Looking at https://google.aip.dev/193, we should be using [google.rpc.Status](https://github.com/googleapis/googleapis/blob/master/google/rpc/status.proto) instead of our custom defined error type. 
Should we drop our error.proto all together? Thoughts? @Linchin @gkcalat